From 0b1ec70bb63adeddcf10c75cb292642a77ebf475 Mon Sep 17 00:00:00 2001
From: Kevin Funk <kfunk@kde.org>
Date: Thu, 11 Aug 2016 11:54:52 +0200
Subject: [PATCH] Restore Windows build

Summary:
xattr is not available on Windows, don't look it up there.
kfilemetadata's xattr_p.h already handles the Windows case without any
API usage of xattr.h

Reviewers: vhanda, sebas, broulik

Differential Revision: https://phabricator.kde.org/D2403
---
 CMakeLists.txt | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4cb67fc..a8c6362 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -61,10 +61,12 @@ set_package_properties(CatDoc PROPERTIES DESCRIPTION "catdoc executable"
                        URL "http://www.wagner.pp.ru/~vitus/software/catdoc/" TYPE RUNTIME
                        PURPOSE "Extract text from office 98 files - RUNTIME dependency")
 
-find_package(Xattr)
-set_package_properties(Xattr PROPERTIES
-    URL "http://savannah.nongnu.org/projects/attr"
-    TYPE REQUIRED)
+if (NOT WIN32)
+    find_package(Xattr)
+    set_package_properties(Xattr PROPERTIES
+        URL "http://savannah.nongnu.org/projects/attr"
+        TYPE REQUIRED)
+endif()
 
 #find_package(QMobipocket)
 #set_package_properties(QMobipocket PROPERTIES DESCRIPTION "Mobipocket epub reader"
